### PR TITLE
Update sandcastle examples to 3DTiles 1.1

### PR DESCRIPTION
**Pull request by jjspace**
# Description

Update the Sandcastle examples under 3DTiles Next to 3DTiles 1.1 including the assets in ion. Also completely removes references to 3DTiles Next in Sandcastle

## Issue number and link

Fixes https://github.com/CesiumGS/cesium/issues/11668

## Testing plan

- Load Sandcastle locally (`npm start`)
- Load each of the examples below in the 3DTiles category and verify they work as expected
  - 3D Tiles 1.1 CDB Yemen
  - 3D Tiles 1.1 Photogrammetry
  - 3D Tiles 1.1 Photogrammetry Classification
  - 3D Tiles 1.1 S2 Globe
  - Custom Shaders 3D Tiles
      - This one uses an Asset Depot asset which will need to be updated as part of a separate effort
  - Custom Shaders Models
  - Custom Shaders Property Textures

# Author checklist

- [x] I have submitted a Contributor License Agreement
- [x] I have added my name to `CONTRIBUTORS.md`
- [ ] I have updated `CHANGES.md` with a short summary of my change
- [ ] I have added or updated unit tests to ensure consistant code coverage
- [ ] I have update the inline documentation, and included code examples where relevant
- [x] I have performed a self-review of my code

### CODE REVIEW

File: Apps/Sandcastle/gallery/3D Tiles 1.1 CDB Yemen.html [hunk 1]
```diff
@@ -9,9 +9,9 @@
     />
     <meta
       name="description"
-      content="Load a 3D Tile Next tileset converted from CDB."
+      content="Load a 3D Tile 1.1 tileset converted from CDB."
```
Comment (ggetz) [inline Apps/Sandcastle/gallery/3D Tiles 1.1 CDB Yemen.html:12]:
```suggestion
      content="Load a 3D Tiles 1.1 tileset converted from CDB."
```

**Review by ggetz — COMMENTED** [review]
Testing is looking good! Just one small suggestion.

**Comment by jjspace** [comment]
@ggetz Updated for the typo

Also updated the Yemen assets again after talking with @lilleyse

**Comment by ggetz** [comment]
> Also updated the Yemen assets again

What was the issue the first go-around?

**Comment by jjspace** [comment]
@ggetz as discussed in person the one I upgraded myself was still using the legacy extensions for some models (`EXT_feature_metadata`). While it worked in CesiumJS there was some concern that it wouldn't in other Cesium Native runtimes.
Even if it would've worked fine updating to the "correctly" generated assets provided by Sean was a small change and easy to make

**Pull request merged**